Example/TextAnnotation/Previews/TextContainerView_Preview.swift

```diff
@@ -25,14 +25,32 @@ struct TextContainerViewPreview: NSViewRepresentable {
   }
 }
 
+struct ExampleScreenshots {
+    static var github = #imageLiteral(resourceName: "github_screenshot")
+    static var landingPage = #imageLiteral(resourceName: "browser_screenshot")
+    static var browserApp = #imageLiteral(resourceName: "browser_app_screenshot")
+    static var figma = #imageLiteral(resourceName: "figma_screenshot")
+}
+
+struct TextParams {
+    var font = "HelveticaNeue-Bold"
+    var fontSize = 30.0
+    var strokeWidth = 3.0
+    var outlineWidth = -2.5
+    var outlineColor = NSColor.white
+    var shadowColor = NSColor.white
+    var shadowOffsetX = 1.5
+    var shadowOffsetY = 1.5
+    var shadowBlur = 2.0
```

Comment:
@ulian-onua I added this stub for a parameter struct that makes clear which options we have.
We should make it easy to wire this up with a settings interface and be able to serialize it.
The current way the parameters are passed in is a bit hard to understand - for example I didn't understand how the font color and separate color param is different.
If we can just use a simple struct like this and make it the public interface of the package that would make it easy to customize.

I'm thinking we should allow passing in an optional params struct like this which can also be used in the Annotations module to serialize to JSON. This way we can easily construct JSON-based examples for different scenarios in the Annotations module.
We might even decide to let Zappy users tweak these parameters in an advanced settings section later.

Let's add these params the public interface of this lib and then move the SwiftUI previews into the Annotations package.